### PR TITLE
Velero: add config for branch oadp-1.2

### PR DESCRIPTION
**Pull request by hhpatel14**

### CODE REVIEW

File: core-services/prow/02_config/openshift/velero/_prowconfig.yaml [hunk 1]
```diff
@@ -13,6 +13,9 @@ branch-protection:
             oadp-1.1:
               allow_force_pushes: true
               protect: true
+            oadp-1.2:
+              allow_force_pushes: true
+              protect: true
 tide:
   queries:
```
Comment (kaovilai) [inline core-services/prow/02_config/openshift/velero/_prowconfig.yaml:20]:
```suggestion
tide:
  merge_method:
    openshift/velero: squash
  queries:
```
Similar to
https://github.com/openshift/release/pull/38010/files#diff-3662ab9ef29d6d8e8c3e3e2bbe09019fad92c59c3e5c3eb488f0984f917d1060R11-R12

**Review by kaovilai — CHANGES_REQUESTED** [review]

File: ci-operator/config/openshift/velero/openshift-velero-oadp-1.2.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
```
Comment (kaovilai) [inline ci-operator/config/openshift/velero/openshift-velero-oadp-1.2.yaml:5]: Per https://github.com/vmware-tanzu/velero/blob/73f17404078f0247c5d27af3890d8f33b598ff6c/Dockerfile#L16
This needs to be 1.19

**Review by kaovilai — CHANGES_REQUESTED** [review]

File: ci-operator/config/openshift/velero/openshift-velero-oadp-1.2.yaml (hunk 2 above)
Comment (kaovilai) [reply ci-operator/config/openshift/velero/openshift-velero-oadp-1.2.yaml:5]:
```suggestion
    tag: golang-1.19
```

**Review by sseago — APPROVED** [review]

**Review by kaovilai — APPROVED** [review]
/lgtm

**Pull request merged**